app/components/modal/add-to-cart/cart.html

```diff
@@ -0,0 +1,26 @@
+<div class="modal-header" style="border-bottom: none; padding-bottom: 0;">
+    <button type="button" class="close" ng-click="cartCtrl.modalCancel()">&times;</button>
+    <h3>{{cartCtrl.order.name | uppercase}}</h3>
+    <h3 style="color: #696969">Rs. {{cartCtrl.order.price}}</h3>
+</div>
+<!--<div class="modal-body">-->
+    <!--<div class="form-group">-->
+        <!--<label for="comment">SPECIAL INSTRUCTIONS</label>-->
+        <!--<textarea placeholder="Add Notes" class="form-control" rows="4" id="comment"></textarea>-->
+    <!--</div>-->
+<!--</div>-->
+<div class="modal-footer" style="border: none;">
+    <div class="pull-left">
+        <button ng-disabled="cartCtrl.count <= 1" class="btn btn-sm" ng-click="cartCtrl.decreaseOrder()">
```

Comment:
use '<==' to compare

Reply:
It throws error. I found JavaScript does not have strict greater/less than comparison operators ;)